### ISSUE TITLE
Feature monitoring: Metrics for cluster nodes

### ISSUE THREAD

**Issue by kushwiz** [vernemq/vernemq#1789]
We do have monitoring for cluster communication in terms of network bytes being transferred.
Can we have similar prometheus metrics for number of cluster nodes ?

similar to output of command:
vmq-admin cluster show

**Comment by ioolkos** [comment]
@kushwiz hm, let's see. So, yes there's no Cluster size metric. Do you mean just that? Cluster size: 3. Note it would be exported from every node, so you'd have 3/3/3 in your Grafana view, which shows what every node thinks about cluster size :)

**Comment by kushwiz** [comment]
@ioolkos yes cluster size :) I spent 2 hours debugging missing MQTT messages and then realized cluster was not formed. Exposing this metric will help me debug this better.
